probatus/feature_elimination/feature_elimination.py

```diff
@@ -960,26 +1001,38 @@ def _get_feature_shap_values_per_fold(self, X, y, clf, train_index, val_index, s
             (np.array, float, float):
                 Tuple with the results: Shap Values on validation fold, train score, validation score.
         """
-        
+
         # The lightgbm imports are temporarily placed here, until the tests on
         # macOS have been fixed for lightgbm.
-        from lightgbm import early_stopping, LGBMModel
-        
+        from lightgbm import early_stopping, log_evaluation, LGBMModel
+
         X_train, X_val = X.iloc[train_index, :], X.iloc[val_index, :]
         y_train, y_val = y.iloc[train_index], y.iloc[val_index]
 
         fit_params = {
-            'X':X_train,
-            'y':y_train,
-            'eval_set':[(X_val, y_val)],
-            'eval_metric':self.eval_metric
+            'X': X_train,
+            'y': y_train,
+            'eval_set': [(X_val, y_val)],
+            'eval_metric': self.eval_metric
         }
 
+        # first_metric_only bypasses a bug that defaults the metric to the
+        # scoring. It should only be True until the bug is found and fixed.
         if isinstance(clf, LGBMModel):
-            fit_params['callbacks'] = [early_stopping(self.early_stopping_rounds)]
+            fit_params['callbacks'] = [
+                early_stopping(
+                    self.early_stopping_rounds, first_metric_only=True
+                )
+            ]
+            
+            if self.verbose == 100:
```

Comment:
Let's do >= 100 so it also works for higher values

Reply:
Makes better sense, thank you!